docker/launch.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z "$FORK" ]]; then
+    FORK=PowerShell
+fi
+
+if [[ -z "$BRANCH" ]]; then
+    BRANCH=master
+fi
+
+# Build both sets by default
+if [[ -z "$BUILDS" ]]; then
+    BUILDS="stable unstable"
+fi
+
+# Build specified distributions
+if [[ -z $DISTROS ]]; then
+    DISTROS="ubuntu14.04 ubuntu16.04 centos7"
+fi
+
+for build in $BUILDS; do
+    # cd so $distro is only the distro name
+    cd $build
```

Comment:
Like I said, it has no error handling :) run it from its folder. I wrote this just to test for starters. It'll need to be made much more robust.